README.md

```diff
@@ -9,13 +12,13 @@
 ### Install via Bower or Node
 
 ```bash
-bower install --save angular-percent-circle-directive
+bower install --save ng-percent-circle-directive
```

Comment:
It's generally bad practice to prefix a custom directive with ng because it can be confused with core angular components. I'll stick with the name I have. I'm going to cancel the pull request you submitted and I will version the project myself now that I am back from vacation.